config.go

```diff
@@ -106,6 +108,27 @@ func (c *Config) Discover() error {
 		}
 	}
 
+	// Finally, we'll fill in any internal plugins that haven't already been
+	// discovered on disk.
```

Comment:
The upgrade process here will be a little weird, since we'll be distributing a single binary to users who have all the plugins on their path.

Do you have any experience from this with Packer? We should figure out what the expected upgrade UX is in this case.